src/main/java/uk/gov/hmcts/cmc/claimstore/events/MoreTimeRequestedCitizenNotificationHandler.java

```diff
@@ -61,12 +67,12 @@ private void sendNotificationToClaimant(final MoreTimeRequestedEvent event) {
         Claim claim = event.getClaim();
 
         Map<String, String> parameters = new HashMap<>();
-        parameters.put("claimReferenceNumber", claim.getReferenceNumber());
-        parameters.put("claimantName", claim.getClaimData().getClaimant().getName());
-        parameters.put("defendantName", claim.getClaimData().getDefendant().getName());
-        parameters.put("responseDeadline", formatDate(event.getNewResponseDeadline()));
-        parameters.put("frontendBaseUrl", notificationsProperties.getFrontendBaseUrl());
-
+        parameters.put(CLAIM_REFERENCE_NUMBER, claim.getReferenceNumber());
```

Comment:
What is the reason behind making such change? Benefits?

Reply:
it's merge gone wrong again

Reply:
You wanted but you did not do it. In current state it does not add any value. What is more is just makes code less readable from the code review perspective. 👎